.documentation.json

```diff
@@ -28,6 +28,9 @@
     "rgba",
     "hsl",
     "hsla",
+    "mix",
+    "hex2rgb",
+    "hex2rgba",
```

Comment:
I think we shouldn't expose hex2rgb & hex2rgba. What's the benefit for the user here? When would she/he use it?